whois-api/src/test/java/net/ripe/db/whois/api/rest/WhoisRestServiceTestIntegration.java

```diff
@@ -4707,6 +4707,33 @@ public void delete_route_out_of_region_redirect() {
         databaseHelper.addAuthoritativeResource("test", "0.0.0.0/0");
     }
 
+    @Test
+    public void delete_domain_trailing_dot_nserver() {
+        databaseHelper.addObject(
+                "domain:        193.in-addr.arpa4\n" +
+                        "nserver:         test.ns.\n" +
+                        "mnt-by:          OWNER-MNT\n" +
+                        "source:          TEST\n");
+
+        final WhoisResources whoisResources = RestTest.target(getPort(), "whois/test/domain/193.in-addr.arpa4")
+                .request()
+                .get(WhoisResources.class);
+        assertThat(whoisResources.getWhoisObjects().get(0).getPrimaryKey().get(0).getValue(), is("193.in-addr.arpa4"));
```

Comment:
Do you need to test the primary key in the test? It seems superfluous

Reply:
Its just a way to test that before delete key exists and after it does not. I can remove that if you prefer that

Reply:
I don't think it's needed, and can be safely removed

Reply:
Done